src/components/views/Profile/Profile.tsx

```diff
@@ -0,0 +1,47 @@
+import auth from '@react-native-firebase/auth';
+import firestore from '@react-native-firebase/firestore';
+import {useActor} from '@xstate/react';
+import {View, Text, Button} from 'react-native';
+import {useContext} from 'react';
+import {GlobalStateContext} from '../../../components/molecules';
+
+export const Profile = () => {
+  const globalServices = useContext(GlobalStateContext);
+  //@ts-ignore
+  const [state, send] = useActor(globalServices.userService);
+  const currentUser = auth().currentUser;
+  const deleteUser = () => {
+    firestore()
```

Comment:
osobna funkcja mate

Reply:
to tak samo jak niżej, taki placeholder jak będę robił Profil to to wypierdolę i po bożemu będzie zrobione